src/Tribe/Service_Providers/Context.php

```diff
@@ -259,21 +259,39 @@ static function ( $data ) {
 				],
 				'tec_post_type'        => [
 					'read' => [
+						Tribe__Context::FUNC          => [
+							static function () {
+								$found = [
+									! empty( tribe_get_request_var( TEC::POSTTYPE, false ) ),
+									! empty( tribe_get_request_var( Venue::POSTTYPE, false ) ),
+									! empty( tribe_get_request_var( Organizer::POSTTYPE, false ) ),
+								];
+								$found = array_filter( $found );
+
+								return count( $found ) ?: Tribe__Context::NOT_FOUND;
+							},
+						],
 						Tribe__Context::LOCATION_FUNC => [
 							'post_type',
 							static function ( $post_type ) {
-								return count(
+								$found = count(
 									array_intersect(
 										(array) $post_type,
-										[ TEC::POSTTYPE, Venue::POSTTYPE, Organizer::POSTTYPE ]
+										[ TEC::POSTTYPE, Venue::POSTTYPE, Organizer::POSTTYPE, ]
 									)
 								);
+								return $found ?: Tribe__Context::NOT_FOUND;
 							},
 						],
 					],
 				],
 				'event_post_type'      => [
 					'read' => [
+						Tribe__Context::FUNC          => [
+							static function () {
+								return ! empty( tribe_get_request_var( TEC::POSTTYPE, false ) ) ?: Tribe__Context::NOT_FOUND;
```

Comment:
:no_entry_sign: Using short ternaries is not allowed

<sub>hash: 3292b9e739d10f509436adcb82ff0897</sub>